Domains/4-Application/Connectors/CivilInfrastructureFramework/CifBridge.ecschema.xml

```diff
@@ -198,10 +217,10 @@
     </ECEnumeration>
     <ECEnumeration typeName="CustomPierBySupportLineRuleAspect_CustomPierBySupportLineRule_LengthAdjustment_Enum" backingTypeName="int" isStrict="true">
         <ECEnumerator name="None" value="0" displayLabel="None"/>
-        <ECEnumerator name="By_Deck" value="1" displayLabel="Deck"/>
-        <ECEnumerator name="By_Skew" value="2" displayLabel="Skew"/>
-        <ECEnumerator name="Centered" value="3" displayLabel="Centered"/>
-        <ECEnumerator name="Deck_And_Centered" value="4" displayLabel="Deck And Aligned"/>
+        <ECEnumerator name="Deck" value="1" displayLabel="Deck"/>
```

Comment:
Changes to Enumerator names are considered major schema updates, so they aren't supported in-place.

These changes were the ones I recall I had to do manually (i.e. keep the name as "By_Deck") since I didn't make the cif2bis tool smart enough to handle them.

Reply:
Good catch! Thanks Diego. I'm going to review the other schemas I changed for this kind of oversight as well. I won't mark this comment as resolved until I've re-reviewed all of them again. Thanks!